frontend/src/theme/components/Checkbox.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentStyleConfig } from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
+
+export const Checkbox: ComponentStyleConfig = {
+  parts: ['others'],
+  baseStyle: ({ theme }) => ({
+    control: {
+      borderRadius: '0.19rem',
```

Comment:
Any reason for this weird rem value? Looked at the figma and it's a 2px radius (i.e 0.125rem)